### PR TITLE
prevent user from sending more than 1 message a second in chat on the frontend

### LINKED ISSUE #152: prevent user from sending more than 1 message a second in chat on the frontend (this is already on backend)

### PR DESCRIPTION
**Pull request by damian-kociszewski**
Fixes #152

### CODE REVIEW

File: kofta/src/vscode-webview/modules/room-chat/RoomChatInput.tsx [hunk 1]
```diff
@@ -63,45 +59,63 @@ export const RoomChatInput: React.FC<ChatInputProps> = ({}) => {
...
+          modalAlert("You got banned from chat");
           return;
-        }
-        if (me.id in useRoomChatStore.getState().bannedUserIdMap) {
-          modalAlert("you got banned from chat");
+      }
+      
+      if (messageTimeout) {
+          if (!toast.isActive("message-timeout")) {
+              toast(
```
Comment (damian-kociszewski) [inline kofta/src/vscode-webview/modules/room-chat/RoomChatInput.tsx:84]: let the user know why the message has not been sent

**Comment by benawad** [comment]
instead of timeout based, can you set the date of the last message sent and check against that

**Comment by nadirabbas** [comment]
@benawad that way we can probably also add timestamps in the chat. maybe in a tooltip.

**Comment by benawad** [comment]
that'll be a separate thing, timestamped on the server

**Comment by damian-kociszewski** [comment]
> instead of timeout based, can you set the date of the last message sent and check against that

sure

**Pull request merged**